src/routes/year/[year]/year.svelte

```diff
@@ -33,518 +40,77 @@
 		['Abmelden', '/logout'],
 		['Impressum', '/impressum']
 	];
+
+	let personPopup: Person | undefined = undefined;
+	function openPersonPopup(event: Event, person: Person) {
+		personPopup = person;
+	}
+
+	function closePersonPopup() {
+		personPopup = undefined;
+	}
 </script>
 
-<Header menu={loggedIn ? headerMenuLoggedIn : headerMenu} />
+<Header menu={data.loggedIn ? headerMenuLoggedIn : headerMenu} />
+<div class="page">
+	<div class="content-wrapper">
+		<HeadlinePage>This is Year {data.displayedYear}</HeadlinePage>
+
+		<Spacer --height="10rem" />
+		<div id="Speaker" class="scroll-anker" />
```

Comment:
Maybe this has to be renamed in multiple places?
```suggestion
		<div id="Speaker" class="scroll-anchor" />
```

Reply:
done